packages/pigeon/platform_tests/test_plugin/ios/Classes/TestPlugin.swift

```diff
@@ -5,6 +5,8 @@
 import Flutter
 import UIKit
 
+extension FlutterError: Error {}
```

Comment:
iiuc this is part of the code that plugin devs write?

Reply:
correct. If this code can be moved to something generated, that would be ideal. It wasn't working when I tried it (though I may have missed something)

Reply:
If you generate multiple codegen files, then you may encounter issue with duplicate extensions, which may be the issue you have. tho I think this here is fine too.